licensing/src/main/java/io/crate/license/DecryptedLicenseData.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Licensed to Crate under one or more contributor license agreements.
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.  Crate licenses this file
+ * to you under the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.  You may
+ * obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied.  See the License for the specific language governing
+ * permissions and limitations under the License.
+ *
+ * However, if you have executed another commercial license agreement
+ * with Crate these terms will supersede the license and you may use the
+ * software solely pursuant to the terms of the relevant commercial
+ * agreement.
+ */
+
+package io.crate.license;
+
+import org.elasticsearch.common.xcontent.XContentBuilder;
+import org.elasticsearch.common.xcontent.XContentFactory;
+import org.elasticsearch.common.xcontent.XContentType;
+
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.util.HashMap;
+import java.util.Objects;
+
+public class DecryptedLicenseData {
+
+    static final String EXPIRATION_DATE_IN_MS = "expirationDateInMs";
+    static final String ISSUED_TO = "issuedTo";
+
+    private final long expirationDateInMs;
+    private final String issuedTo;
+
+    DecryptedLicenseData(long expirationDateInMs, String issuedTo) {
+        this.expirationDateInMs = expirationDateInMs;
+        this.issuedTo = issuedTo;
+    }
+
+    long expirationDateInMs() {
+        return expirationDateInMs;
+    }
+
+    String issuedTo() {
+        return issuedTo;
+    }
+
+    /*
+     * Creates the json representation of the license information with the following structure:
+     *
+     * <pre>
+     *      {
+     *          "expirationDateInMs": "XXX",
+     *          "issuedTo": "YYY"
+     *      }
+     * </pre>
+     */
+    byte[] formatLicenseData() {
+        try {
+            XContentBuilder contentBuilder = XContentFactory.contentBuilder(XContentType.JSON);
+            HashMap<String, Object> licenseContent = new HashMap<>(2);
```

Comment:
The intermediate map could probably be avoided here by using the `field` methods on the XContentBuilder